ci/scripts/install-standalone-vm.bash

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/env bash
+
+# script that installs teet app, postgresql db, datomic local, postgrest api on one vm.
+#
+# usage:
+# run this from shell on your dev machine:
+# ci/scripts/install-standalone-vm.bash launchvm <ssh-key-name>
+# where ssh key name a key name from the registered ec2 key pairs in the aws env.
+# This will start the vm, detect what git branch you are on and try to check out that same branch at the remote vm.
+
+
+MACHINE_ID=b # pick one from a-e that doesn't conflict with someone else
```

Comment:
how should this be picked?

Reply:
ah, I thought I was passing this in as an env var, will fix.